src/screens/HomeScreen.js

```diff
@@ -29,36 +31,197 @@ const styles = StyleSheet.create({
     textAlign: "center",
     color: "#333333",
     marginBottom: 5
+  },
+  tabContainer: {
+    backgroundColor: variables.palette.gray.uiBackground,
+    width: "100%",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    paddingHorizontal: 10,
+    height: "100%",
+    alignItems: "center",
+    marginTop: 10,
+    height: 40,
+    flexWrap: "wrap",
+    marginBottom: 20
+  },
+  imageTabContainer: {
+    backgroundColor: variables.palette.gray.uiBackground,
+    width: "100%",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    paddingHorizontal: 10,
+    height: "100%",
+    alignItems: "center",
+    marginTop: 10,
+    height: 40,
+    flexWrap: "wrap",
+    marginBottom: 100,
+  },
+  list: {
+    flexDirection: "row",
+    flexWrap: "wrap"
+  },
+  bottomBodyPart: {
+    marginTop:20,
+    marginRight: 50,
+    marginLeft:5
+  },
+  profileBodyPartTop: {
+    marginTop:20,
+    marginRight: 50,
+    marginLeft:5,
+    marginBottom:50
+  },
+  topBodyPart: {
+    height:75,
+    width:100,
+    marginRight: 25,
+    marginLeft:25
+  },
+  profilePic: {
+    width: 150,
+    height: 300,
+    marginTop:20,
+    marginRight: 25,
+    marginLeft:25
+  },
+  profileBodyParts: {
+    width: 60,
+    height: 60,
+    marginTop:80,
+    marginRight: 50,
+    marginLeft:45,
   }
 });
 
 
 class HomeScreen extends React.Component {
   constructor(props) {
     super();
+    this.showImageFunc = this.showImageFunc.bind(this);
+    this.showEyeImageFunc = this.showEyeImageFunc.bind(this);
+    this.showMouthImageFunc = this.showMouthImageFunc.bind(this);
+    this.selectTopImage = this.selectTopImage.bind(this);
     this.state = {
-      editMode: false
+      editMode: false,
+      showImage: false,
+      showEyeImage: false,
+      showMouthImage: false,
+      topImage: 0,
+      midImage:0,
+      bottomImage:0,
+      topImageFinal: 0,
+      midImageFinal:0,
+      bottomImageFinal:0,
+      hair : null,
+      eyes : null,
+      mouth : null,
+      hairTextColour : "#333333",
+      eyeTextColour: "#333333",
+      mouthTextColour: "#333333",
+      Top : [
+        {
+          "image": require("../assets/avatar/top/brain.png")
```

Comment:
These images can be moved to the Constants.json

Reply:
Thanks, done!